torch/csrc/api/src/nn/modules/dropout.cpp

```diff
@@ -15,6 +20,10 @@ variable_list Dropout::forward(variable_list inputs) {
   return lst;
 }
 
+Dropout2d::Dropout2d(double p) : p_(p) {
+  assert(p < 1 && p >= 0);
```

Comment:
Just FYI, asserts will usually compile to no-ops. We should use `TORCH_ASSERT` for checks

Reply:
Yes will change that. Btw we should maybe canonicalize `AT_CHECK`  and `TORCH_ASSERT` ? I think only using `AT_CHECK` everywhere would make sense. @ezyang